ios/CameraView+TakePhoto.swift

```diff
@@ -103,7 +103,13 @@ extension CameraView {
         photoSettings.isAutoContentAwareDistortionCorrectionEnabled = enableAutoDistortionCorrection
       }
 
-      photoOutput.capturePhoto(with: photoSettings, delegate: PhotoCaptureDelegate(promise: promise))
+      // disableShutterSound
+      var disableShutterSound = false
+      if let _disableShutterSound = options["disableShutterSound"] as? Bool{
+        disableShutterSound = _disableShutterSound
+      }
```

Comment:
This probably looks a bit cleaner as a one-liner? not sure though
```suggestion
      let disableShutterSound = (options["disableShutterSound"] as? Bool) ?? false
```